soundcalc/zkvms/fri_based_vm.py

```diff
@@ -89,7 +89,7 @@ class FRIBasedCircuitConfig:
     # Preset field parameters (contains p, ext_size, F)
     field: FieldParams
     # Total columns of AIR table
-    num_columns: int
+    num_columns: list[int]
```

Comment:
Kinda confused on what is a list of `num_columns`. I would expect number of columns of the AIR trace to stay static within a FRI run. Is this implying something else? Let's document what it means.